packages/keycloak-user-management/src/constants.tsx

```diff
@@ -29,6 +29,6 @@ export const PRACTITIONER_UPDATED_SUCCESSFULLY = 'Practitioner updated successfu
 export const NO_DATA_FOUND = 'No Data Found';
 
 // rendered text
-export const CREDENTIALS = 'Credentials';
-export const RESET_PASSWORD = 'Reset Password';
+export const CREDENTIALS = 'User Credentials';
+export const RESET_PASSWORD = 'Set password';
```

Comment:
Why are we making this change?

Reply:
The credentials piece doesn't only do a reset. It does the setting of the password too when the user is created.